src/libs/OptionsListUtils.js

```diff
@@ -449,6 +450,8 @@ function createOption(logins, personalDetails, report, reportActions = {}, {show
 
         if (result.isChatRoom || result.isPolicyExpenseChat) {
             result.alternateText = showChatPreviewLine && !forcePolicyNamePreview && lastMessageText ? lastMessageText : subtitle;
+        } else if (result.isMoneyRequestReport) {
+            result.alternateText = lastMessageTextFromReport || Localize.translate(preferredLocale, 'report.noActivityYet');
```

Comment:
```suggestion
            result.alternateText = lastMessageTextFromReport.length > 0 ? lastMessageText : Localize.translate(preferredLocale, 'report.noActivityYet');
```

Reply:
Also update https://github.com/Expensify/App/blob/0af44b3efad7f4b1035a723f30dacb24aea0adae/src/libs/OptionsListUtils.js#L434

to

```js
lastMessageTextFromReport = report ? report.lastMessageText || '' : '';
```


Logic taken from SidebarUtils for consistency.

Reply:
Actually this may not be required. Depends if money request report can multiple participants.